### PR TITLE
dev: line messaging api update on June 2020

### PR DESCRIPTION
**Pull request by hch903**
LineClient.ts getGroupSummary and getGroupMembersCount / getRoomMembersCount function (TODO: test case)

### CODE REVIEW

File: packages/messaging-api-line/src/LineClient.ts [hunk 1]
```diff
@@ -1697,6 +1741,25 @@ export default class LineClient {
...
+   *
+   * @param roomId - Room ID. Found in the `source` object of [webhook event objects](https://developers.line.biz/en/reference/messaging-api/#webhook-event-objects).
+   * @returns Returns status code `200` and a JSON object with the following information.
+   *
+   * count:
+   * - The count of members in the group. The number returned excludes the LINE Official Account.
+   */
+  getRoomMembersCount(roomId: string): Promise<Types.MemberCount> {
+    return this._axios
+      .get(`/v2/bot/group/${roomId}/members/count`)
```
Comment (etrex) [inline packages/messaging-api-line/src/LineClient.ts:1759]: the path should be `/v2/bot/room/${roomId}/members/count`

File: packages/messaging-api-line/src/LineClient.ts [hunk 2]
```diff
@@ -1697,6 +1741,25 @@ export default class LineClient {
...
+   * @param roomId - Room ID. Found in the `source` object of [webhook event objects](https://developers.line.biz/en/reference/messaging-api/#webhook-event-objects).
+   * @returns Returns status code `200` and a JSON object with the following information.
+   *
+   * count:
+   * - The count of members in the group. The number returned excludes the LINE Official Account.
+   */
+  getRoomMembersCount(roomId: string): Promise<Types.MemberCount> {
+    return this._axios
+      .get(`/v2/bot/room/${roomId}/members/count`)
+      .then((res) => res.data, handleError);
```
Comment (chentsulin) [inline packages/messaging-api-line/src/LineClient.ts:1760]: Could we get the count number directly here using `res.data.count`?

```js
await getRoomMembersCount(ID);
// => 3
```

File: packages/messaging-api-line/src/LineClient.ts [hunk 3]
```diff
@@ -1630,6 +1630,50 @@ export default class LineClient {
...
+   * @param groupId - Group ID. Found in the `source` object of [webhook event objects](https://developers.line.biz/en/reference/messaging-api/#webhook-event-objects).
+   * @returns Returns status code `200` and a JSON object with the following information.
+   *
+   * count:
+   * - The count of members in the group. The number returned excludes the LINE Official Account.
+   */
+  getGroupMembersCount(groupId: string): Promise<Types.MemberCount> {
+    return this._axios
+      .get(`/v2/bot/group/${groupId}/members/count`)
+      .then((res) => res.data, handleError);
```
Comment (chentsulin) [inline packages/messaging-api-line/src/LineClient.ts:1674]: Could we get the count number directly here using `res.data.count`?

```js
await getGroupMembersCount(ID);
// => 3
```

File: packages/messaging-api-line/src/LineClient.ts [hunk 4]
```diff
@@ -1630,6 +1630,50 @@ export default class LineClient {
...
+   * [Official document - get members in group count](https://developers.line.biz/en/reference/messaging-api/#get-members-group-count)
+   *
+   * @param groupId - Group ID. Found in the `source` object of [webhook event objects](https://developers.line.biz/en/reference/messaging-api/#webhook-event-objects).
+   * @returns Returns status code `200` and a number representing group member count.
+   *
+   * count:
+   * - The count of members in the group. The number returned excludes the LINE Official Account.
+   */
+  getGroupMembersCount(groupId: string): Promise<Types.MemberCount> {
+    return this._axios
```
Comment (chentsulin) [inline packages/messaging-api-line/src/LineClient.ts:1672]: @hch903 Please rebase the master branch and rename `this._axios` to `this.axios`.

**Comment by hch903** [comment]
Sorry, I messed up the git log...

**Pull request closed without merging**